retrieval/scrape.go

```diff
@@ -74,12 +74,29 @@ var (
 		},
 		[]string{interval},
 	)
+	targetSyncIntervalLength = prometheus.NewSummary(
```

Comment:
Jobs are few enough that aggregating over them doesn't add much. Detecting slow ones here by eye is sufficient.